From 41ed20b9703027dfd7c644099bdbe7a9de3775f5 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>, Peter Michael Green <plugwash@debian.org>
Date: Sat, 5 Feb 2022 19:54:40 -0500
Subject: [PATCH 2/2] Relax dependency on bindgen
Last-Updated: Sat, 12 Aug 2023 21:50:43 +0200 werdahias@riseup.net
Forwarded: not-needed

This dependency is designed by upstream to keep their MSRV low, but
debian doesn't need to worry about that (the distro's MSRV is managed
differently).

However due to Debian bug #967954 the upper limit on the build-dependency in
Debian does need to be restricted to not allow upstream minor versions greater
than the one currently in Debian.
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: nettle-sys/Cargo.toml
===================================================================
--- nettle-sys.orig/Cargo.toml
+++ nettle-sys/Cargo.toml
@@ -42,7 +42,7 @@ dependencies = [
 version = "0.2"
 
 [build-dependencies.bindgen]
-version = ">= 0.58.0, < 0.64.0"
+version = "0.66"
 features = ["runtime"]
 default-features = false
 
